src/Enterspeed.Cli/Commands/Schema/ShowDeployedSchemasCommand.cs

```diff
@@ -0,0 +1,61 @@
+﻿using System.CommandLine;
+using System.CommandLine.Invocation;
+using Enterspeed.Cli.Api.Environment;
+using Enterspeed.Cli.Api.MappingSchema;
+using Enterspeed.Cli.Services.ConsoleOutput;
+using MediatR;
+using Microsoft.Extensions.Logging;
+
+namespace Enterspeed.Cli.Commands.Schema
+{
+    public class ShowDeployedSchemasCommand : Command
```

Comment:
We haven't used the term "show" before. In the other commands we use the term "list" instead. So we also use list instead of show here?

Reply:
Changes has been implemented @jesperweber